src/app/settings/views/Security/SessionTimeout/SessionTimeout.test.tsx

```diff
@@ -0,0 +1,133 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import configureStore from "redux-mock-store";
+
+import SessionTimeout, {
+  Labels as SessionTimeoutLabels,
+} from "./SessionTimeout";
+
+import { actions as configActions } from "app/store/config";
+import type { RootState } from "app/store/root/types";
+import { renderWithBrowserRouter, getTestState } from "testing/utils";
+
+const mockStore = configureStore<RootState>();
+
+describe("SessionTimeout", () => {
+  let state: RootState;
+
+  beforeEach(() => {
+    state = getTestState();
+  });
+
+  it("displays a spinner while loading", () => {
+    state.config.loaded = false;
+    state.config.loading = true;
+    renderWithBrowserRouter(<SessionTimeout />, { state });
+
+    expect(screen.getByText(SessionTimeoutLabels.Loading)).toBeInTheDocument();
+  });
+
+  it("displays the form when the 'Edit' button is clicked", async () => {
```

Comment:
This form isn't enabled on click.

```suggestion
  it("displays the form with correct values", async () => {
```